MANIFEST.in

```diff
@@ -15,11 +15,14 @@ include *.sh
 include *.yaml
 include *.yml
 
+exclude pytask.ini
+
 graft docs
```

Comment:
I think the docs should not be part of the package. Nobody is looking for them in ``site-packages/estimagic`` anyways.

Reply:
Will postpone until I improve packaging in general.